### ISSUE TITLE
Manaan Hrakert Station - Inconsistent dialogue with mercenary

### ISSUE THREAD

**Issue by Salk73** [KOTORCommunityPatches/K1_Community_Patch#378]
When the party travels via sub to the Hrakert Station, they meet a mercenary just behind the first door. The dialogue is triggered by opening that door and after talking to him, without ending up fighting him, it is possible to talk to him again right afterwards.

If you do, this is what he says:

"Where did you go? How did you get back here without being eaten?"

The problem here is that E1 has a conditional script (_k_pman_hrak01_) that checks if the global boolean MAN_HRAKERT_DONE is set to 1. If it is, it resets it to zero and speaks that line. This global boolean is set to 1 after the player is arrested coming out of the Sith Embassy.

That is obviously wrong.

What should happen is that the variable should be set to 1 upon leaving the Hrakert Station instead.

**Comment by DarthParametric** [comment]
> This global boolean is set to 1 after the player is arrested coming out of the Sith Embassy.

Yes, for some reason it is set in the Sith base's OnExit script `k_pman_27_areae`. Not sure what the story is there. Seems like it must be an error of some kind. Looks like `MAN_SITHBASE_SEALED` gets set TRUE at the trial, so it presumably it wasn't meant to be that. I see `MAN_SITHB1_DONE` and  `MAN_SITHB2_DONE` are still both FALSE in post-base/post-trial saves, so I don't know if those are relevant or not. 

I'd suggest it should be changed to a different check altogether, either going deeper into the station, or going back to the surface (since there is a specific line for that). Not sure if that would require the creation of new booleans though.
